source/fundamentals/connection/connection-options.txt

```diff
@@ -208,8 +208,20 @@ parameters of the connection URI to specify the behavior of the client.
 
    * - **maxConnecting**
      - integer
-     - Specifies the number of connections the driver will driver concurrently
-       establish before making them available in the connection pool.
+     - Specifies the maximum number of connections a pool may be establishing
+       concurrently. Defaults to ``2``.
+
+   * - **srvServiceName**
+     - string
+     - Specifies the service name the driver retrieves from your SRV record to
+       construct your 
+       :manual:`seed list </reference/glossary/#std-term-seed-list>`.
```

Comment:
Thank you for the detailed comment.

## Updated Understanding

SRV resource records are key value pairs that bind `(service, protocol, domain)` tuples to `(target domain, port tuples)`. *inference to test understanding*: as multiple seed list entries correspond to one SRV resource record key, SRV resource record keys are not unique. 

## Discussion of Suggested Text

- I believe "overrides the default" can be simplified to "specifies". As the documentation aims to be accessible to [Jasmine Persona readers](https://drive.google.com/file/d/14FbBOLCVxwSP6M9BK4Nz1Ir9tzxT8_02/view), we try limit technical language whenever possible.
- I believe the default value "`mongodb`" should be specified at the end of the option. I believe allows the page to be more scannable by readers, as they do not have to read into the description of options to find the default value.
- As specifying the option does not change the service name of the SRV resource record key stored on DNS, should the "overrides the SRV resource records..." be updated to "specified the service name of the SRV resource records the driver retrieves..." (emphasize that you are specifying what the driver retrieves rather than what is stored on DNS).

## Updated Suggested Text

TODO: link to changes

Reply:
> _Inference to test understanding_: as multiple seed list entries correspond to one SRV resource record key tuple, SRV resource record key tuples are not unique.

Right, they are not unique. Also, while we and RFC 2782 may conceptually treat them as triples `(service, protocol, domain)`, from the perspective of DNS they are just names of records (`_service._protocol.domain`), with each dot-separated part (`_service`, `_protocol`, `domain`) being an opaque label. Names of records of other types may also be non-unique, for example, multiple A records with the same name may be used when the same domain is mapped to multiple IP addresses, see [round-robin DNS](https://www.cloudflare.com/en-ca/learning/dns/glossary/round-robin-dns/).